18F/identity-idp
# Change robots to deny by default with allowlist

## 🛠 Summary of changes

Updates `robots.txt` to disallow all by default, with few exceptions. Also removes redundant `robots` `<meta>` tag.

Related Slack discussion: https://gsa-tts.slack.com/archives/C0NGESUN5/p1711054927329839

**Why?**

- The previous `robots.txt` was incomplete and allowed crawling beyond what we would expect to be allowed
- Since there are very few pages we expect to be crawlable, it's easier to list them explicitly
- By preventing most all pages from being crawled, it's no longer necessary to also include a `meta` directive preventing indexing ([related resource](https://support.google.com/webmasters/thread/152254598?hl=en&msgid=152256454))

**Open questions:**

- Trailing slash? Technically both `/en/` and `/en` are valid. We tend to link to the one without the final trailing slash, and it seems reasonable to only want to allowing crawling of one and not both versions of the page.

## 📜 Testing Plan

1. Visit http://localhost:3000/robots.txt
2. Verify entries match expected crawlable routes
